### PR TITLE
Fix #53

### LINKED ISSUE #53: Blog post order is confusing
The time after the dot shows last modified, but sorted by created time.

image

### PR DESCRIPTION
**Pull request by colourdelete**

### CODE REVIEW

File: core/templates/core/blogpost/card_snippet.html [hunk 1]
```diff
@@ -19,10 +19,10 @@ <h1 class="title">{{blogpost.title}}</h1>
...
                     {{blogpost.author.username|user}}
                     <br>
-                    <dot>•</dot> {{ blogpost.last_modified_date|date:"M. j, Y" }}{{ blogpost.last_modified_date|date:", g:i A"|lower }}
-                    <!--{% if blogpost.last_modified_date != blogpost.created_date %}
-                    (Edited)
-                    {% endif %}-->
+                    <dot>•</dot> posted {{ blogpost.created_date|date:"M. j, Y" }}{{ blogpost.created_date|date:", g:i A"|lower }}
+                    {% if blogpost.last_modified_date != blogpost.created_date %}
+                    (Edited on {{ blogpost.created_date|date:"M. j, Y" }}{{ blogpost.created_date|date:", g:i A"|lower }})
+                    {% endif %}
```
Comment (hillcrestpaul0719) [inline core/templates/core/blogpost/card_snippet.html:25]: I feel like this should be commented out still & be changed to last_modified

Comment (colourdelete) [reply core/templates/core/blogpost/card_snippet.html:25]: OK

Comment (colourdelete) [reply core/templates/core/blogpost/card_snippet.html]: Does this LGTY?

Comment (hillcrestpaul0719) [reply core/templates/core/blogpost/card_snippet.html]: I think you shouldn't delete but just comment it out /shrug
Also comment out the entire if statement including `(Edited)` display

Comment (colourdelete) [reply core/templates/core/blogpost/card_snippet.html:26]: Done!

Comment (hillcrestpaul0719) [reply core/templates/core/blogpost/card_snippet.html:26]: tysm! (sorry for miscommuniation)

**Pull request merged**